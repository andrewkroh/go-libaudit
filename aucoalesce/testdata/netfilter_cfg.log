type=NETFILTER_CFG msg=audit(1492648520.473:4678): table=filter family=2 entries=7
type=SYSCALL msg=audit(1492648520.473:4678): arch=c000003e syscall=54 success=yes exit=0 a0=4 a1=0 a2=40 a3=562bf7914100 items=0 ppid=9708 pid=9709 auid=4294967295 uid=0 gid=0 euid=0 suid=0 fsuid=0 egid=0 sgid=0 fsgid=0 tty=(none) ses=4294967295 comm="iptables" exe="/sbin/xtables-multi" key=(null)
type=PROCTITLE msg=audit(1492648520.473:4678): proctitle=2F7362696E2F69707461626C6573002D77002D49007373686775617264002D730034352E37362E3134352E313532002D6A0044524F50
type=EOE msg=audit(1492648520.473:4678):