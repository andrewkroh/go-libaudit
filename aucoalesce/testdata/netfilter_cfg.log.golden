{
  "@timestamp": "2017-04-20T00:35:20.473Z",
  "sequence": 4678,
  "category": "configuration",
  "record_type": "netfilter_cfg",
  "result": "success",
  "session": "unset",
  "actor": {
    "primary": "unset",
    "secondary": "0",
    "attrs": {
      "egid": "0",
      "euid": "0",
      "fsgid": "0",
      "fsuid": "0",
      "gid": "0",
      "sgid": "0",
      "suid": "0"
    }
  },
  "thing": {
    "primary": "filter"
  },
  "action": "loaded-firewall-rule-to",
  "data": {
    "a0": "4",
    "a1": "0",
    "a2": "40",
    "a3": "562bf7914100",
    "arch": "x86_64",
    "comm": "iptables",
    "entries": "7",
    "exe": "/sbin/xtables-multi",
    "exit": "0",
    "family": "2",
    "pid": "9709",
    "ppid": "9708",
    "proctitle": "/sbin/iptables",
    "syscall": "setsockopt",
    "table": "filter",
    "tty": "(none)"
  }
}
